.kitchen.yml

```diff
@@ -17,4 +17,10 @@
           name: terraform
           root_module_directory: examples/sample-jhub
           command_timeout: 12000
-          parallelism: 1
\ No newline at end of file
+          parallelism: 1
+      # - name: sample-jhub-nfs
+      #   driver:
+      #     name: terraform
+      #     root_module_directory: examples/sample-jhub-nfs
+      #     command_timeout: 12000
+      #     parallelism: 1
```

Comment:
Do you need this commented out stuff?

Reply:
I don't :) but I wanted as a reminder that it's not enabled..  I can remove it though - I have an issue for re-enabling nfs testing